packages/openapi-fetch/test/fixtures/e2e/app.ts

```diff
@@ -0,0 +1,52 @@
+import createClient from "../../../src";
+import type { paths } from "./e2e.d.ts";
+
+const client = createClient<paths>({
+  baseUrl: "/api/v1",
+});
+
+/**
+ * Test 1: GET /api/v1/get
+ */
+async function testGet() {
```

Comment:
New tests that get loaded in a true browser environment. Not comprehensive, but we at least have:

- GET request (no body)
- POST request (with body)
- POST request with multi-part/form data